aten/src/ATen/cpu/vec256/vec256_bfloat16.h

```diff
@@ -252,6 +252,15 @@ template <> class Vec256<BFloat16> {
   Vec256<BFloat16> expm1() const {
     return map(Sleef_expm1f8_u10);
   }
+  Vec256<BFloat16> fmod(const Vec256<BFloat16> & q) const {
```

Comment:
Is the BFloat16 fmod specialization used somewhere? I saw only the dispatch code in BinaryOpsKernel.cpp for `kHalf` but not for `kBFloat16`?

Reply:
It's not currently used, but since it's a floating point type it should have that operation available. I can also enable `BFloat16` dispatch if that's desirable.